dlc-manager/tests/ln_dlc_channel_execution_tests.rs

```diff
@@ -951,6 +951,29 @@ fn ln_dlc_rejected_offer() {
     reject_offer(&test_params);
 }
 
+
+#[test]
+#[ignore]
+fn ln_dlc_rejected_offer_then_new_offer_and_accept_offer() {
+    let test_params = test_init();
+    make_ln_payment(&test_params.alice_node, &test_params.bob_node, 900000);
+
+    reject_offer(&test_params);
+
+    offer_sub_channel_internal(&test_params, false);
```

Comment:
nit: can you use `open_sub_channel` instead since it's there?

Reply:
done: https://github.com/p2pderivatives/rust-dlc/pull/182/commits/5f80d389a038baa3d4210625cfb1a63f3a3e7c4b